content/en/mobile/_index.md

```diff
@@ -83,11 +83,120 @@ On the Incidents page, you can view, search and filter all incidents that you ha
 
 ## Widgets
 
+### Open Incidents widget
+
+{{< img src="mobile/incident_widget.png" alt="widget page" responsive="true" style="width:100%; background:none; border:none; box-shadow:none;">}}
+
+View your [open incidents][13] from your mobile home screen with Datadog widgets.
+
+You can tap any open incident displayed in the widget to have it open with more details in the Datadog mobile app - perfect for diving deeper into issues. 
+
+Also, you can customize your Open Incidents widgets by filtering on:
+- Organization
+- Serverity levels
+- Customers impacted
+- Ordering
+
+#### Create an Open Incidents widget
+
+{{< tabs >}}
+{{% tab "iOS" %}}
+
+- Long press on your home screen.
+- Tap the “+” button on the top left corner of the screen.
+- Search for "Datadog" widgets.
+- Select your prefered size (small, medium, or large).
+- Drag the widget to your desired, on-screen location.
+
+
+{{% /tab %}}
+{{% tab "Android" %}}
+
+- Long press on your home screen.
+- Tap the "Widgets" button on your home screen editor. If you have app shortcuts, it might appear as just an icon on the top right corner of the bubble.
+- Drag the widget to your desired, on-screen home screen.
+- Resize the widget to fit your preference.
```

Comment:
```suggestion
1. Long press on your home screen.
2. Tap the **Widgets** button on your home screen editor. If you have app shortcuts, it might appear as only an icon on the top right corner of the bubble.
3. Drag the widget to your desired, on-screen home screen.
4. Resize the widget to fit your preference.
```